specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2020-08-04-preview/AlertsManagement.json

```diff
@@ -0,0 +1,649 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RuleNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to delete an health alert rule"
+          },
+          "204": {
+            "description": "No content: the request was successful, but the response is empty"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/HealthAlertsErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeleteResourceHealthAlertRule": {
+            "$ref": "./examples/deleteResourceHealthAlert.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "HealthAlertAction": {
+      "description": "An alert action.",
+      "properties": {
+        "actionGroupId": {
+          "type": "string",
+          "description": "the id of the action group to use."
+        },
+        "webHookProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "the dictionary of custom properties to include with the post operation. These data are appended to the webhook payload."
+          },
+          "description": "The properties of a webhook object."
+        }
+      }
+    },
+    "HealthAlertProperties": {
+      "required": [
+        "enabled",
+        "description",
+        "criteria"
+      ],
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "the description of the health alert that will be included in the alert email."
+        },
+        "enabled": {
```

Comment:
strongly encourage using "alertState": "Enabled"/"Disabled" (or similar) with a string enum instead to allow for future states

Reply:
Thanks @pilor - we would like this to be aligned with current metric alerts API in order not to cause confusion with customers...